components/quentn/sources/campaign-contact-sent/campaign-contact-sent.mjs

```diff
@@ -0,0 +1,21 @@
+import app from "../../quentn.app.mjs";
+
+export default {
+  key: "quentn-campaign-contact-sent",
+  name: "New Campaign Contact Sent",
+  description: "Emit new event when a campaign contact is sent.",
```

Comment:
Does the webhook need to be set up in the app's UI? Maybe include a link or a description about how to set that up?